### PR TITLE
docs: improve 'Introduction'

### PR DESCRIPTION
**Pull request by hassankhan**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

Minor improvements to Introduction page

### CODE REVIEW

File: docs/content/Cube.js-Introduction.md [hunk 1]
```diff
@@ -1,41 +1,75 @@
...
-__Cube.js is an open source modular framework to build analytical web applications__. It is primarily used to build internal business intelligence tools or to add customer-facing analytics to an existing application.
+**Cube.js is an open-source modular framework to build analytical web
+applications**. It is primarily used to build internal business intelligence
+tools or to add customer-facing analytics to an existing application.
 
-Cube.js was designed to work with Serverless Query Engines like AWS Athena and Google BigQuery. Multi-stage querying approach makes it suitable for handling trillions of data points. Most modern RDBMS work with Cube.js as well and can be tuned for adequate performance.
+Cube.js was designed to work with Serverless Query Engines like AWS Athena and
+Google BigQuery. A multi-stage querying approach makes it suitable for handling
+trillions of data points. Most modern RDBMS work with Cube.js as well and can
+further be tuned for performance.
```
Comment (rpaik) [inline docs/content/Cube.js-Introduction.md:16]:
```suggestion
be further tuned for performance.
```

File: docs/content/Cube.js-Introduction.md [hunk 2]
```diff
@@ -1,41 +1,75 @@
...
+If you are building your own business intelligence tool or customer-facing
+analytics, it is quite likely you'll face one or more of the following problems:
 
-1. __Performance.__ Most of effort time in modern analytics software development is spent to provide adequate time to insight. In the world where every company data is a big data writing just SQL query to get insight isn't enough anymore.
-2. __SQL code organization.__ Modelling even a dozen of metrics with a dozen of dimensions using pure SQL queries sooner or later becomes a maintenance nightmare which ends up in building modelling framework.
-3. __Infrastructure.__ Key components every production-ready analytics solution requires: analytic SQL generation, query results caching and execution orchestration, data pre-aggregation, security, API for query results fetch, and visualization.
+1. **Performance.** A significant amount of time in modern analytics software
+   development is spent providing adequate time to insight. In a world where
+   every companies' data is big data, writing just SQL queries for insights
```
Comment (rpaik) [inline docs/content/Cube.js-Introduction.md:44]:
```suggestion
   every companies' data is big data, just writing SQL queries for insights
```

File: docs/content/Cube.js-Introduction.md [hunk 3]
```diff
@@ -1,41 +1,75 @@
...
 ## Why Cube.js?
 
-If you are building your own business intelligence tool or customer-facing analytics most probably you'll face following problems:
+If you are building your own business intelligence tool or customer-facing
+analytics, it is quite likely you'll face one or more of the following problems:
 
-1. __Performance.__ Most of effort time in modern analytics software development is spent to provide adequate time to insight. In the world where every company data is a big data writing just SQL query to get insight isn't enough anymore.
-2. __SQL code organization.__ Modelling even a dozen of metrics with a dozen of dimensions using pure SQL queries sooner or later becomes a maintenance nightmare which ends up in building modelling framework.
-3. __Infrastructure.__ Key components every production-ready analytics solution requires: analytic SQL generation, query results caching and execution orchestration, data pre-aggregation, security, API for query results fetch, and visualization.
+1. **Performance.** A significant amount of time in modern analytics software
```
Comment (rpaik) [inline docs/content/Cube.js-Introduction.md:42]: This first sentence is a bit awkward (esp. with the word "time" used twice). How about "A significant effort in modern analytics software development is on providing adequate time to insight"?

File: docs/content/Cube.js-Introduction.md [hunk 4]
```diff
@@ -1,41 +1,75 @@
...
+   dimensions using pure SQL queries eventually becomes a maintenance nightmare,
+   which then requires engineering effort in building a modelling framework
+3. **Infrastructure.** Any production-ready analytics solution requires key
+   components such as analytic SQL generation, query results caching and
+   execution orchestration, data pre-aggregation, security, API for query
+   results fetch, and visualization
 
-Cube.js has necessary infrastructure for every analytic application that heavily relies on its caching and pre-aggregation layer to provide several minutes raw data to insight delay and sub second API response times on a trillion of data points scale.
+Cube.js has the necessary infrastructure for every analytic application that
+heavily relies on its caching and pre-aggregation layer to provide several
```
Comment (rpaik) [inline docs/content/Cube.js-Introduction.md:55]:
```suggestion
relies heavily on its caching and pre-aggregation layer to provide several
```

File: docs/content/Cube.js-Introduction.md [hunk 5]
```diff
@@ -1,41 +1,75 @@
...
+minutes raw data to insight delay and sub second API response times on a
```
Comment (rpaik) [inline docs/content/Cube.js-Introduction.md:56]: I think the word "delay" sounds negative? 
How about "to provide insight from raw data in minutes and sub-second API response times on a trillion data points"?

Comment (hassankhan) [reply docs/content/Cube.js-Introduction.md:56]: Yeah this whole sentence was a bit iffy ... wasn't entirely sure how to reword it. I like your suggestion, I've slightly added to it:

> to provide insights from raw data within minutes and an API with sub-second response times on up to a trillion data points

**Review by igorlukanin — APPROVED** [review]
Mostly formatting, right? LGTM 👍

**Comment by hassankhan** [comment]
@igorlukanin Mostly formatting, but includes some general fixes to structure, incorrect grammar etc.

**Pull request merged**